docs/ref/databases.txt

```diff
@@ -577,7 +580,7 @@ Here's a sample configuration which uses a MySQL option file::
     database = NAME
     user = USER
     password = PASSWORD
-    default-character-set = utf8
+    default - character - set = utf8
```

Comment:
I think this code a different code block from the previous.

Reply:
Fixed by marking it setting it to have `ini` file pygment:

```
.. code-block:: ini
```